index.php

```diff
@@ -25,9 +25,9 @@
     error_reporting(-1);
 }
 
-// Ensure PHP version >= 7.4
-if (PHP_VERSION_ID < 70400) {
-    die('NamelessMC is not compatible with PHP versions older than 7.4, you are running PHP ' . PHP_VERSION);
+// Ensure PHP version >= 8.2
```

Comment:
Do we really want to kill every site that still uses 7.4? A warning that stuff Might break is probably less intrusive than immediately nuking okder sites? Regardless, how will this work with the updater?

Reply:
yeah that sounds like a better idea, don't want the site to just suddenly break when updating.

Reply:
Yeah great idea. We can update the releases API to display the required PHP version, then we can display a nice big red error if we detect a mismatch locally